Adjust/example/src/main/java/com/adjust/example/GlobalApplication.java

```diff
@@ -83,6 +85,18 @@ public void onFinishedSessionTrackingFailed(AdjustSessionFailure sessionFailureR
             }
         });
 
+        // evaluate deeplink to be launched
```

Comment:
This listener is synchronous, so the sdk is waiting until the execution and it is running on the background.
Should it run in the main thread and be async?